packages/leavitt-elements/src/event-bus.ts

```diff
@@ -57,6 +57,10 @@ export class EventBus<TEntityTypes, TEventTypes> {
   }
 
   dispatch<TArg>(entityType: TEntityTypes, eventTypes: TEventTypes, object?: TArg) {
-    this.#subscribers.filter(s => s.entityType === entityType && (s.eventTypes === 'All' || s.eventTypes === eventTypes)).forEach(s => s.callback(object));
```

Comment:
I see a few issues here: 
- This will break existing app where we rely on subscribers to be notified prior to the next code being ran
- This could cause batch updates in lit to no longer be batched

Reply:
Points one and two completely defeat the entire point of using a subscriber pattern. If we _are_ doing those anywhere we should not be. If you are relying on something to happen before moving on, it should be called directly, not like this. If you are worried about the data mutating, then - however it's being called - you should be passing a snapshot of the data, not a reference to the object. Which is what we do almost everywhere.

I also, don't see how this could have any significant impact on batched updates. From my understanding, updates are batched by each component, individually. The whole batching system is basically just a next-tick debouncer on requestUpdate. The only case I can think of where this would actually impact it, is if the component is subscribed to an event that it is, itself, dispatching. Which we probably shouldn't do.